recipes/pyhacrf-datamade/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "pyhacrf-datamade" %}
+{% set version = "0.2.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyhacrf-datamade-{{ version }}.tar.gz
+  sha256: cf9dc239090696cee301d5d32074e1ed881853c6da789ce05817c072b22c4f44
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
```

Your error is because `pylbfgs` is not available on Windows. Since this is not noarch Python, you can just skip the build on Windows:
```suggestion
  skip: true  # [win]
  script: {{ PYTHON }} -m pip install . -vv
```